stackslib/src/clarity_vm/clarity.rs

```diff
@@ -141,6 +141,31 @@ pub struct ClarityTransactionConnection<'a, 'b> {
     epoch: StacksEpochId,
 }
 
+impl<'a, 'b> ClarityTransactionConnection<'a, 'b> {
+    pub fn new(
```

Comment:
Where is this getting used? If we're going to define this method, then please use it in stackslib/src/clarity_vm/clarity.rs and give it a more descriptive name.

Reply:
Ok, thanks. I was originally thinking that the name could indicate something about the log setup that it's doing, but since it seems everyone is doing that, maybe it's fine to just be `new`.